### PR TITLE
docs: add open source templates to  resources page

### PR DESCRIPTION
**Pull request by marqbeniamin**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [x] Other... Please describe: docs: add 2 open source templates to  resources page

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/marketing/resources.json [hunk 1]
```diff
@@ -519,6 +519,16 @@
             "desc": "Taiga UI is fully-treeshakable Angular UI Kit consisting of multiple base libraries and several add-ons. We have 130+ components, 100+ directives, dozens of tokens, utils and tools. They are easily customizable and well engineered.",
             "title": "Taiga UI",
             "url": "https://taiga-ui.dev"
+          },
+          "mdct": {
+            "desc": "Angular Material Dashboard  is a Free and Open Source Material Bootstrap Admin with a fresh, new design inspired by Google's Material Design.",
```
Comment (twerske) [inline aio/content/marketing/resources.json:524]:
```suggestion
            "desc": "Angular Material Dashboard is a Free and Open Source Material Bootstrap Admin with a fresh, new design inspired by Google's Material Design.",
```

**Review by twerske — APPROVED** [review]
lgtm with nit fixes please!

**Comment by AndrewKushnir** [comment]
@marqbeniamin could you please also rebase this PR on top of the latest master branch? That'd trigger CI to rerun with the most up-to-date configuration, which would likely resolve the `legacy-unit-tests-saucelabs` CI job failure. Thank you.

**Comment by alxhub** [comment]
(closing due to inactivity - @marqbeniamin feel free to reopen once you rebase this)

**Pull request closed without merging**